qa/rpc-tests/test_framework/blocktools.py

```diff
@@ -74,6 +73,8 @@ def create_transaction(prevtx, n, sig, value, scriptPubKey=CScript()):
     tx.vin.append(CTxIn(COutPoint(prevtx.sha256, n), sig, 0xffffffff))
     tx.vout.append(CTxOut(value, scriptPubKey))
     tx.calc_sha256()
```

Comment:
This should come after the padding step.

Reply:
Thanks :+1:

Reply:
Needs fix